source/grid/cell_id.cc

```diff
@@ -168,6 +168,14 @@ CellId::to_cell(const Triangulation<dim, spacedim> &tria) const
   return cell;
 }
 
+
+
+types::coarse_cell_id
```

Comment:
I think this now needs to be removed as you inlined it.

Reply:
I have inlined it.